apigw-lambda-snapstart/UnicornFunction/src/main/java/com/unicorn/store/StreamLambdaHandler.java

```diff
@@ -7,13 +7,20 @@
 import com.amazonaws.serverless.proxy.spring.SpringBootProxyHandlerBuilder;
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.lambda.powertools.logging.Logging;
+import software.amazon.lambda.powertools.metrics.Metrics;
+import software.amazon.lambda.powertools.tracing.Tracing;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
 public class StreamLambdaHandler implements RequestStreamHandler {
```

Comment:
Also, once M2 is released we won't need this class at all and instead use the new `SpringDelegatingLambdaContainerHandler` as demonstrated here https://github.com/awslabs/aws-serverless-java-container/tree/main/samples/springboot3/alt-pet-store

Reply:
I would be happy to raise a pr to to update this when m2 is out